gulp/gulpfile.js

```
@@ -26,80 +26,89 @@ var current_config = {
 
 function prepare_revealjs() {
     return src(current_config.nodeModulesDir + '/reveal.js/dist/**/*')
-        .pipe(dest(current_config.buildDir + '/reveal.js/dist/'));
+        .pipe(dest(current_config.buildDir + '/reveal.js/dist/'))
+        .pipe(browserSync.stream());;
```

Biome

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)